docs/source/release_notes.rst

```diff
@@ -1,6 +1,23 @@
 Release Notes
 -------------
-**Future Release**
+**Future Releases**
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. warning::
+
+    **Breaking Changes**
+
+
+**v0.30.2 Aug. 16, 2021**
+    * Fixes
+        * Updated changelog and version numbers to match the release.
```

Comment:
Yep. For ultimate clarity please mention that v0.30.1 was erroneously released without a release PR and that v0.30.2 replaces it.